src/app/app-modules/nurse-doctor/visit-details/upload-files/upload-files.component.ts

```diff
@@ -346,7 +346,8 @@ export class UploadFilesComponent implements OnChanges, OnInit, DoCheck {
           (res: any) => {
             if (res.data.statusCode === 200) {
               const fileContent = res.data.data.response;
-              location.href = fileContent;
+              // location.href = fileContent;
```

Comment:
remove the commented code if not needed anymore

Reply:
done